diff -ur rspamd-2.3/src/libutil/util.c rspamd-2.3.cipher/src/libutil/util.c
--- rspamd-2.3/src/libutil/util.c	2020-02-04 11:00:17.000000000 -0500
+++ rspamd-2.3.cipher/src/libutil/util.c	2020-03-10 15:30:10.936313892 -0400
@@ -2436,7 +2436,7 @@
 rspamd_config_libs (struct rspamd_external_libs_ctx *ctx,
 		struct rspamd_config *cfg)
 {
-	static const char secure_ciphers[] = "HIGH:!aNULL:!kRSA:!PSK:!SRP:!MD5:!RC4";
+	static const char secure_ciphers[] = "PROFILE=SYSTEM";
 	size_t r;
 	gboolean ret = TRUE;
 
